my_game.py

```diff
@@ -58,22 +57,51 @@ def __init__(self, center_x, center_y, lives, scale):
         # Graphics to use for Player
         super().__init__("images/playerShip1_red.png")
 
-        self.speed = 1
```

Comment:
let's keep self.speed. Multiply the result from cos() and sin() with it (See comment below).